DLCS.Repository/DlcsContext.cs

```diff
@@ -451,7 +451,12 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
                 entity.Property(e => e.Id).HasMaxLength(500);
 
-                entity.Property(e => e.Aliases).HasMaxLength(1000);
+                entity.Property(e => e.Aliases)
+                    .HasMaxLength(1000);
+                    // We have to roll this back because we have a Dapper mapping for this

```

Comment:
Is it worth fixing dapper, or removing this comment? Trying to avoid 18months from now this comment still being there!

Looking at deliverator I'm not sure this field is ever used anywhere other than mapping back and forth to/from db/hydra